sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -79,7 +79,7 @@ public async Task BuildDocumentModel(DocumentBuildMode buildMode)
             var prefix = "subfolder";
             var options = new BuildDocumentModelOptions()
             {
-                Description = $"This model was generated by a .NET test: {nameof(BuildDocumentModel)}",
+                Description = $"This model was generated by a .NET test.",
```

Comment:
Making the model description the same across different tests to make more use of the newly implemented model cache.